index.js

```diff
@@ -16,25 +15,23 @@ module.exports.invoke = (name, payload) => {
 		return Promise.reject(new Error('Please provide a valid function name'));
 	}
 
-	const params = {
+	const parameters = {
 		FunctionName: parsed.functionName,
 		InvocationType: 'RequestResponse',
 		Payload: JSON.stringify(payload)
 	};
 
 	if (parsed.qualifier) {
-		params.Qualifier = parsed.qualifier;
+		parameters.Qualifier = parsed.qualifier;
 	}
 
-	return pify(this.raw.invoke.bind(this.raw))(params)
+	return lambda.invoke(parameters).promise()
 		.then(data => {
 			let payload = data.Payload;
 
 			try {
 				payload = JSON.parse(payload);
-			} catch (err) {
-
-			}
+			} catch { }
```

Comment:
I don't think this will work. I believe it works since Node.js 12 or something. So use `catch (_) {` instead.